src/indexExample2.ts

```diff
@@ -61,9 +61,15 @@ const contents = fs.readFileSync(inputFilePath, {
 const countsByLibrary = JSON.parse(contents) as Record<string, number>;
 const repositoryIdentifiers = Object.keys(countsByLibrary);
 
-const labels = ['good first issue'];
+const labels = ['good first issue', 'help wanted', 'documentation'];
 const finder = new IssueFinder(logger);
 finder
     .findIssues(token, labels, repositoryIdentifiers)
-    .then((issues) => logger.info(`Found ${issues.length} issues \n ${JSON.stringify(issues)}`))
-    .catch((err) => logger.error(err.message));
+    .then((issues) => {
+        logger.info(`Found ${issues.length} issues \n`);
+        // logger.info(`${JSON.stringify(issues)}`);
+    })
+    .catch((err) => {
+        logger.error(err.message);
+        console.log(err);
```

Comment:
As mentioned, it would be nice to be able to log the stack trace through the logger variable, but since we don't yet have that capability, I'm OK leaving it like this in indexExample2.